components/ErrorDisplay/ErrorDisplay.js

```diff
@@ -8,6 +8,7 @@ ErrorDisplay.propTypes = { statusCode: PropTypes.number.isRequired };
 function ErrorDisplay({ statusCode }) {
   return (
     <div className={styles.ErrorDisplay}>
+      <meta name="robots" content="noindex, nofollow" />
```

Comment:
Instead of inlining it, can you use the Next.js `<Head/>` component so that we dont lose SSR/CSR parity?